libraries/stumbler/src/main/java/org/mozilla/mozstumbler/svclocator/services/log/DebugLogger.java

```diff
@@ -28,7 +31,10 @@ public String e(String logTag, String s, Throwable e) {
         if (e == null) {
             msg = "";
         } else {
-            msg = e.toString();
+            StringWriter sw = new StringWriter();
```

Comment:
I reverted the DebugLogger to spew full stacktraces when exceptions are passed into the Log.e handler.

Note that this only affects *debug* builds and not release builds.